firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -43,7 +43,12 @@ class TranslationSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = dictionary.Translation
-        fields = ("text", "language", "part_of_speech")
+        fields = ("id", "text", "language", "part_of_speech")
```

Comment:
(Optional for consistency) Can also inherit DictionaryContentMeta and add the 2 extra fields there e.g.
`fields = DictionaryContentMeta.fields + ("language", "part_of_speech")`